### PR TITLE
git-contrib: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-contrib.md
Comment (navarroaxel) [inline pages/common/git-contrib.md:9]: `username` is a word:
```suggestion
`git contrib {{username}}`
```

Comment (bl-ue) [reply pages/common/git-contrib.md:9]: But is it referring to e.g. the GitHub handle (`navarroaxel`) or the user's display name (`Axel Navarro`)?

Comment (navarroaxel) [reply pages/common/git-contrib.md:9]: Based on the `git log --author` you can use name, full name, username or email.

File: pages/common/git-contrib.md
Comment (marchersimon) [inline pages/common/git-contrib.md:7]:
```suggestion
- Display all commit hashes and corresponding commit messages from a specific author:
```
`from a specific username` sounds weird and it's already implied that every author has to have a usermane.

File: pages/common/git-contrib.md
Comment (bl-ue) [inline pages/common/git-contrib.md:7]:
```suggestion
- Display all commit hashes and their corresponding commit messages from a specific author:
```

**Review by marchersimon — APPROVED** [review]

File: pages/common/git-contrib.md
Comment (navarroaxel) [inline pages/common/git-contrib.md:9]: Based on the `git-log.md` page, we used `author` as a placeholder for `first name`, `full name`, `email` or `username`:
```suggestion
`git contrib {{author}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
@bl-ue just waiting for your approval then we'll merge :)

**Review by bl-ue — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Thanks

**Pull request merged**
